utility_network/perform-valve-isolation-trace/README.md

```diff
@@ -0,0 +1,55 @@
+# Perform valve isolation trace
+
+Run a filtered trace to locate operable features that will isolate an area from the flow of network resources.
+
+![Image of Perform valve isolation trace](PerformValveIsolationTrace.png)
+
+## Use case
+
+Determine the set of operable features required to stop a network's resource, effectively isolating an area of the network. For example, you can choose to return only accessible and operable valves: ones that are not paved over or rusted shut.
+
+## How to use the sample
+
+Create and set the configuration's filter barriers by selecting a category. Check or uncheck 'Include Isolated Features'. Click 'Trace' to run a subnetwork-based isolation trace.
+
+## How it works
+
+1. Create an `ArcGISMap` and add it to a `MapView`.
+2. Using the URL to a utility network's feature service, create `FeatureLayer`s that contain the utility network's features, and add them to the operational layers of the map.
+3. Create and load a `UtilityNetwork` with the same feature service URL and map.
+4. Create a `UtilityElement` to represent a default starting location from the `UtilityNetwork`, using a given asset type and global id.
+5. Add a `GraphicsOverlay` with a `Graphic` that represents this starting location.
+6. Use `utilityNetwork.fetchFeaturesForElementsAsync()` to obtain the geometry of this element, and place the created graphic at that geometry.
```

Comment:
I'd swap steps 5 and 6 over, since the call to `utilityNetwork.fetchFeaturesForElementsAsync` comes immediately after creating the UtilityElement representing the starting location.